scio-examples/src/main/java/org/apache/beam/examples/common/ExampleCloudSqlOptions.java

```diff
@@ -0,0 +1,46 @@
+package org.apache.beam.examples.common;
+
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.Validation;
+
+public interface ExampleCloudSqlOptions extends PipelineOptions {
```

Comment:
Should this be included as ready to use `CloudSqlOptions` that get registered if `scio-jdbc` is included? You can take a look at how `ScioOptions` are defined and registered in `scio-core`.